coalib/bears/Bear.py

```diff
@@ -460,14 +460,17 @@ def execute(self, *args, debug=False, **kwargs):
                 self.err('Bear {} violated one-based offset convention.'
                          .format(name), str(exc))
 
-            if self.kind() == BEAR_KIND.LOCAL:
-                self.err('Bear {} failed to run on file {}. Take a look '
-                         'at debug messages (`-V`) for further '
-                         'information.'.format(name, args[0]))
+            if self.kind() == BEAR_KIND.LOCAL
+            and self.section['log_level'].value != 'DEBUG':
```

Comment:
Also if we think about your logic it says
If the Debug is not set then print error else print error. So its gonna print the error in both the situations.